fedimint-api/src/core/server.rs

```diff
@@ -395,12 +397,13 @@ where
                 handler: Box::new(
                     move |module: &DynServerModule,
                           dbtx: fedimint_api::db::DatabaseTransaction<'_>,
-                          value: serde_json::Value| {
+                          value: serde_json::Value,
+                          module_instance_id: Option<ModuleInstanceId>| {
```

Comment:
I don't think we need to add this arg here in `api_endpoints` and could just inject the appropriate module id into the DB tx in.

https://github.com/fedimint/fedimint/blob/2c75860db88e0903e40a4ebd6c9f263553435a09/fedimint-server/src/net/api.rs#L133-L180